auth_proxy/models/oauth.py

```diff
@@ -21,8 +21,11 @@ class Client(Base):
     client_id = Column(String, primary_key=True)
     client_secret = Column(String, nullable=False)
 
-    _redirect_uris = Column(Text)
-    _default_scopes = Column(Text)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    user = orm.relationship('User')
```

Comment:
But what I'm saying is that I'd expect many *users* for one *client* -- for example, there will be one PMI app registered with a given FHIR API endpoint. And many different users will all approve that single client. Does this make sense?